movinglater/gotools/add_inside_block.go

```diff
@@ -65,3 +74,34 @@ func findClosingRouteBlockEnd(search string, f *ast.File, fset *token.FileSet, f
 
 	return end
 }
+
+func findBlockCoordinates(search string, f *ast.File, fset *token.FileSet, fileLines []string) (int, int) {
```

Comment:
You could probably change this signature to take a padded file. It would make it easier for others to use. :)